modules/logging-collector-alerts.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * logging/logging_alerts/default-logging-alerts.adoc
+
+:_content-type: REFERENCE
+[id="logging-collector-alerts_{context}"]
+= Logging collector alerts
+
+In logging 5.8 and later versions, the following alerts are generated by the {clo}. You can view these alerts in the {product-title} web console.
```

Comment:
"logging" her is used a verb when it shoul, I think, be {clo}? Or maybe the "Logging 5.8 implementation"?

Reply:
"logging" is the approved short name for the logging product, this usage is correct though I see where you're coming from.